e2e/cypress/e2e/translations/mtSettings.cy.ts

```diff
@@ -55,7 +55,7 @@ describe('Machine translation settings', () => {
     );
   });
 
-  it('will update language specific settings', () => {
+  it('will update language specific settings', { retries: 2 }, () => {
```

Comment:
Is this the retries 2 intended? If it's really fleaky and it's unrsolvable, I would allow 5 retrials, otherwise, let's remove it.